src/utils/metadataUtils.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-param-reassign */
+const metadataKeyToName = (key) => key.replace('-', ' ');
+
+const metadataNameToKey = (name) => `${name.trim().replace(/\s+/g, '-')}`;
```

Comment:
Why do we need this transformation? I am under the impression that JSON places no limitation on what we can use as a key. Maybe it is something that we'll need for DynamoDB?

Reply:
Yes, you're right, there is no limitation on the keys. However, just to keep things tidy and easier to read, the key names are transformed.